src/lib/stepper/_stepper-theme.scss

```diff
@@ -20,18 +21,31 @@
     }
 
     .mat-step-icon {
+      color: mat-color($primary, default-contrast);
+    }
+
+    .mat-step-icon-selected {
       background-color: mat-color($primary);
       color: mat-color($primary, default-contrast);
     }
 
-    .mat-step-icon-not-touched {
+    .mat-step-icon-not-selected {
       background-color: mat-color($foreground, disabled-text);
       color: mat-color($primary, default-contrast);
     }
 
+    .mat-step-icon-error {
+      background-color: transparent;
+      color: mat-color($warn);
+    }
+
     .mat-step-label.mat-step-label-active {
       color: mat-color($foreground, text);
     }
+
+    .mat-step-label.mat-step-label-error {
```

Comment:
Can the combined class selector be avoided? In general we try to keep our selector specificity as low as possible.

Reply:
In this case, it cannot, just like how it could not be avoided in the `.mat-step-label.mat-step-label-active` case as well as in the other cases in this `scss` file.